test/hs/Test/Ganeti/HTools/Node.hs

```diff
@@ -134,6 +134,7 @@ genEmptyOnlineNode =
                        , Node.rMemForth = 0
                        , Node.pRem = 0
                        , Node.pRemForth = 0
+                       , Node.xMem = 0
```

Comment:
Can we completely get rid of xMem? It is calculated from other measured values, so this could be replaced by a function.